source/type/pb_converter.cpp

```diff
@@ -1946,6 +1946,7 @@ pbnavitia::Metadatas* PbCreator::mutable_metadatas(){
 
 void PbCreator::clear_feed_publishers(){
     response.clear_feed_publishers();
+    contributors.clear();
```

Comment:
The clear you point is to not duplicate contributors if you call get_response several times. It's done after importing the contributors in the pb feedpublisher. This clear just remove the contributors. When you'll call get_response, contributors will then be empty. That was the bug, we was cleaning the pb but the pb was not filled.